documentation/man/features/init.rst

```diff
@@ -29,6 +29,12 @@ OPTIONS
   Set the variable `default_deploy_target` from **kworkflow.config** to
   *<target>*, which can be any of vm, local or remote.
 
+\--interactive:
```

Comment:
This is missing the `-i` option. Also, you need to add this option to the synopsis.

Reply:
Look at `build.rst` for reference.